plugins/ExamplePlugin/changes.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "title":       "New feature x added",
+    "description": "Now you can do a with b like this",
+    "linkName":    "For more information go here",
```

Comment:
Should that be `linkName` or `link_name`? In PHP code you are looking for `link_name`, therefor this change would be added without a link...

Reply:
Good spot, fix committed.